public/javascripts/search-bar.js

```diff
@@ -0,0 +1,15 @@
+const searchBar = document.forms['search-box'].querySelector('input')
+
+searchBar.addEventListener('keyup', function (e) {
+	const term = e.target.value.toLowerCase()
+	const cards = document.getElementById('card-list').children
+
+	Array.from(cards).forEach(function (card) {
+		const title = card.firstElementChild.textContent
+		if (title.toLowerCase().indexOf(term) != -1) {
+			card.style.display = 'block'
```

Comment:
Prefer adding / removing a class over doing inline styles. Separation of concerns

Reply:
This will also ensure you can more easily animate in/out based on css transitions and animations